.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - copy-gradle-properties
       - android/restore-gradle-cache
       - android/publish-to-s3:
-          module_name: WordPressUtils
+          publish_gradle_task: :WordPressUtils:publishLibraryToS3
```

Comment:
My only question is about the double `: :` but I know Gradle uses that for things internally.

If it's ok with you, it's good with me!

Reply:
When you say double `:`, I assume you're referring to the ones in `:WordPressUtils:publishLibraryToS3`.

This is actually the correct syntax and my understanding is that `WordPressUtils:publishLibraryToS3` [will not be a valid command in Gradle `7.0`](https://github.com/Automattic/publish-to-s3-gradle-plugin/pull/11#issuecomment-843533705).